x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/common.tsx

```diff
@@ -57,11 +58,11 @@ export const buildColumns: (showFooter: boolean) => Array<EuiBasicTableColumn<Ta
     sortable: true,
     dataType: 'number',
     align: 'right',
-    render: (score: number) => displayNumber(score),
+    render: (score: number) => formatRiskScore(score),
```

Comment:
is this extra arrow function needed? we can just pass the format function, no?

Reply:
It is unnecessary, indeed. I am deleting it. Thank you!